llvm/test/MC/WebAssembly/reference-types.s

```diff
@@ -4,22 +4,27 @@
 # CHECK-LABEL:ref_is_null:
 # CHECK: ref.is_null     # encoding: [0xd1]
 ref_is_null:
-  .functype ref_is_null () -> (i32, i32)
+  .functype ref_is_null () -> (i32, i32, i32)
   ref.null_extern
   ref.is_null
   ref.null_func
   ref.is_null
+  ref.null_exn
+  ref.is_null
```

Comment:
I don't see any CHECK expectations for this (or for null func). I assume this is just because it's sufficient to check the encoding once and the rest is to make sure the type check passes?

Reply:
It only had one check line before:
https://github.com/llvm/llvm-project/blob/99835922ca2a2ac20769271a49a5f8055bb5dc93/llvm/test/MC/WebAssembly/reference-types.s#L5

I guess it was written that way because the encoding for `ref.is_null` is the same regardless of what type it takes. And encodings for `ref.null_[func/extern/exn]` is checked in `ref_null_test` below.